analysis/new_solved_net_by_week.sql

```diff
@@ -18,6 +21,8 @@ solved_tickets as
 (
 	select solved_week as week, count(*) as num_solved
 	from ticket_summary
+	-- make sure the ticket is actually solved
+	where solved_week is not null
 	group by solved_week
 )
 
```

Comment:
`cum_new` and `cum_solved` should actually just be new and solved. backlog should be cumulative, but the new and solved lines should be flows for that given week. just remove the window function for each of them.